app/controllers/chatrooms_controller.rb

```diff
@@ -22,4 +23,9 @@ def my_messages
   def chatroom_params
     params.permit(:name)
   end
+
```

Comment:
when you open a given chatroom, you will set the notifications for that specific chatroom from unread status to read.